frontend/services/gui/src/GUIManager.cpp

```diff
@@ -1149,18 +1149,23 @@ void GUIManager::draw_menu() {
             float scale = megamol::gui::gui_scaling.Get();
             if (ImGui::RadioButton("100%", (scale == 1.0f))) {
                 this->SetScale(1.0f);
+                ImGui::CloseCurrentPopup();
```

Comment:
I am not really sure I want that. Good question actually.

Reply:
Updated slightly in following commit to fit UX of other menus.